jerry-core/ecma/operations/ecma-eval.cpp

```diff
@@ -47,6 +47,11 @@ ecma_op_eval (ecma_string_t *code_p, /**< code string */
   ecma_completion_value_t ret_value;
 
   lit_utf8_size_t chars_num = ecma_string_get_size (code_p);
+  if (chars_num == 0)
+  {
+    ret_value = ecma_make_normal_completion_value (ecma_make_simple_value (ECMA_SIMPLE_VALUE_UNDEFINED));
+    return ret_value;
+  }
   MEM_DEFINE_LOCAL_ARRAY (code_utf8_buffer_p,
```

Comment:
@idkiller, could you, please, remove the `return` on line 53 and put the rest of the function until line 72 to an `else` block?